lib/parse-args.js

```diff
@@ -154,6 +170,10 @@ function buildYargs (withCommands = false) {
     })
     .pkgConf('c8')
     .demandCommand(1)
+    // Todo:  These next 6 lines might need to get removed
+    // This logic is covered with a default argument in
+    // the config options declaration in this file for yargs
+    /* c8 ignore next 6 */
     .check((argv) => {
       if (!argv.tempDirectory) {
         argv.tempDirectory = resolve(argv.reportsDir, 'tmp')
```

Comment:
From my understanding, this function should not actually change arguments but only checks to see if they are valid, per the [yargs documentation.](https://yargs.js.org/docs/#api-reference-checkfn-globaltrue)

This condition was never reached per the former code coverage report.  Added a coverage ignore comment line. 

Setting the tempDirectory when empty should be addressed in this patch on lines 128 -135.

I suggest these lines can be deleted.